src/scripts/components/stories/pilot-story/components/satellite-carousel/satellite-carousel.tsx

```diff
@@ -0,0 +1,100 @@
+import React, {useEffect, useState} from 'react';
+
+import {Parallax} from 'react-scroll-parallax';
+import {useSwipeable} from 'react-swipeable';
+
+import {satellitesByName} from '../../types/globe';
+import SatelliteElement from '../satellite-element/satellite-element';
+
+import {useScreenSize} from '../../../../../hooks/use-screen-size';
+
+import styles from './satellite-carousel.module.styl';
+
+const SatelliteCarousel = () => {
+  const [selectedIconIndex, setSelectedSourceIndex] = useState(0);
+
+  const {isMobile} = useScreenSize();
+
+  const satellites = Object.entries(satellitesByName).map(([key, value]) => ({
+    key,
+    value
+  }));
+
+  const [windowHeight, setWindowHeight] = useState(window.innerHeight);
+
+  const diameter = windowHeight * 0.8;
+
+  const radius = diameter / 2;
```

Comment:
You could `useMemo()` here to only recalculate values if `windowHeight` or diameter have `changed`